src/Confluent.Kafka/Admin/AlterUserScramCredentialsResult.cs

```diff
@@ -0,0 +1,44 @@
+// Copyright 2023 Confluent Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Refer to LICENSE for more information.
+
+using System;
+using System.Collections.Generic;
+
+namespace Confluent.Kafka.Admin
+{
+    /// <summary>
+    ///     The result for an alter user scram credentials request
+    ///     including errors.
+    /// </summary>
+    public class AlterUserScramCredentialsResult
+    {
+        /// <summary>
+        ///     Reports for each user scram credentials alterations
+        /// </summary>
+        public List<AlterUserScramCredentialsReport> AlterUserScramCredentialsReports { get; set;}
```

Comment:
Don't use `AlterUserScramCredentialsReports` in `AlterUserScramCredentialsResult`. Rather add all the fields again here, except Error